internal/util/rocksmq/server/rocksmq/rocksmq_reader.go

```diff
@@ -38,8 +38,8 @@ type rocksmqReader struct {
 //Seek seek the rocksmq reader to the pointed position
 func (rr *rocksmqReader) Seek(msgID UniqueID) { //nolint:govet
 	rr.currentID = msgID
-	fixChanName, _ := fixChannelName(rr.topic)
-	dataKey := path.Join(fixChanName, strconv.FormatInt(msgID, 10))
+	fixTopicName := rr.topic + "/"
```

Comment:
Do we need to ban topicname contains "/" ?

Reply:
I'll add this constraint in CreateTopic in later pr.